package.json

```diff
@@ -57,5 +57,8 @@
     "mocha": "^2.2.5",
     "rimraf": "^2.4.2",
     "uglifyjs": "^2.4.10"
+  },
+  "dependencies": {
+    "isarray": "^1.0.0"
```

Comment:
It's supported for the browsers that this module supports: http://kangax.github.io/compat-table/es5/#test-Array_methods_Array.isArray